lib/pages/namespace/namespace_edit.dart

```diff
@@ -97,10 +104,13 @@ class _NamespaceEditPageState extends State<NamespaceEditPage> {
     // FIXME: is there a way we can update the namespace without creating a new namespace object?
     //  aka updating the existing namespace we got from context (setters?)
     Namespace updatedNamespace = Namespace(
-        id: widget.namespace.id,
-        title: _name,
-        description: _description,
-        owner: widget.namespace.owner);
+      id: widget.namespace.id,
+      title: _name,
+      description: _description,
+      owner: widget.namespace.owner,
+      created: widget.namespace.created,
+      updated: widget.namespace.updated,
+    );
```

Comment:
We should either make title and description of Namespace non-final or add a copyWidth function.

Reply:
Agreed. I'll do the copyWith function for the moment, but I think we need to have a larger discussion on whether we want to have mutable or immutable models. I am personally leaning towards immutable as that makes the most sense to me especially with Flutter widgets being immutable. However, neither Dart nor Flutter are opinionated on the subject for data models.